Neos.Neos/Documentation/References/NeosFusionReference.rst

```diff
@@ -744,9 +744,8 @@ into rendering a page; responsible for rendering the ``<html>`` tag and everythi
 :body.templatePath: (string) Path to a fluid template for the page body
```

Comment:
This isn't needed anymore.

Reply:
correct adjusted ... why is this before the head documentation